### PR TITLE
Add stats

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: src/translations/en/stats.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "instances.dating.desc": "Dating services used in Europe, in English and in the jurisdictions of the European Union and Switzerland.",
+  "instances.dating.subtitle": "• 24 services <br/>• 52 documents",
+  "instances.dating.title": "Dating",
+  "instances.title": "Our data is produced by instances maintained by various partners to cover different industries, languages and jurisdictions",
+  "numbers.block1.desc": "spread over industries, languages and jurisdictions",
+  "numbers.block1.title": "400+ services",
+  "numbers.block2.desc": "classified along a taxonomy of over fifty types",
+  "numbers.block2.title": "900+ documents",
+  "numbers.block3.desc": "published in the form of short stories and tweets",
+  "numbers.block3.title": "200+ analyses",
```
Comment (martinratinaud) [inline src/translations/en/stats.json:24]:
```suggestion
  "numbers.block3.title": "200+ analysis",
```

File: src/translations/en/stats.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+**Open Terms Archive's mission is to shift the power balance from large platforms to their users and regulators.** Measuring the specific impact of Open Terms Archive in such a broad task is difficult, if not impossible. Despite these limitations, we seek to measure the effect of our efforts, both to maximise it and out of accountability to the entities that trust us.
+
+## Qualitative impact
+
+Much of the impact of Open Terms Archive is inherently unmeasurable. In this section we refer to prominent cases that illustrate those effects that make sense when contextualised and lose meaning when reduced to a number, as their systemic impact is not linear with their occurrence.
+
+### Bills
+
+Open Terms Archive supported the [TLDR Act](https://www.lifewire.com/the-tldr-act-could-help-you-make-sense-of-terms-of-service-agreements-5216643) (for _Terms-of-service Labeling, Design, and Readability_), a US bill co-sponsored by Congresswoman Lori Trahan (D-MA-03), Senator Bill Cassidy (R-LA), and Senator Ben Ray Luján (D-NM) to enforce the publication of summaries of terms of use in machine-readable formats.
```
Comment (martinratinaud) [inline src/translations/en/stats.mdx:11]:
```suggestion
Open Terms Archive supported the [TLDR Act](https://www.lifewire.com/the-tldr-act-could-help-you-make-sense-of-terms-of-service-agreements-5216643) (for _Terms-of-service Labeling, Design, and Readability_), a US bill co-sponsored by [Congresswoman Lori Trahan](https://trahan.house.gov/) (D-MA-03), [Senator Bill Cassidy](https://www.cassidy.senate.gov/) (R-LA), and Senator Ben Ray Luján (D-NM) to enforce the publication of summaries of terms of use in machine-readable formats.
```

File: src/translations/fr/stats.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "instances.contrib.author": "Contributions bénévoles",
+  "instances.contrib.desc": "Services divers en anglais.",
+  "instances.contrib.subtitle": "• 285 services <br/>• 612 documents",
+  "instances.contrib.title": "Contrib",
+  "instances.france.author": "UFC Que Choisir | Ambassadeur français pour le numérique",
+  "instances.france.desc": "Services divers en français et dans la juridiction française.",
+  "instances.france.subtitle": "• 104 services <br/>• 206 documents",
+  "instances.france.title": "France",
+  "instances.france-elections.author": "Reset",
+  "instances.france-elections.desc": "Services pouvant influencer les élections présidentielle et législative françaises 2022, en français et dans la juridiction française.",
```
Comment (martinratinaud) [inline src/translations/fr/stats.json:11]:
```suggestion
  "instances.france-elections.desc": "Services pouvant influencer les élections présidentielles et législatives françaises 2022, en français et dans la juridiction française.",
```

File: src/translations/fr/stats.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  "instances.dating.author": "HestiaLabs",
+  "instances.dating.desc": "Services de rencontres utilisés en Europe, en anglais et dans les juridictions de l'Union européenne et de la Suisse.",
+  "instances.dating.subtitle": "• 24 services <br/>• 52 documents",
+  "instances.dating.title": "Dating",
+  "instances.title": "Nos données sont produites par des instances maintenues par divers partenaires pour couvrir différentes industries, langues et juridictions",
+  "numbers.block1.desc": "répartis sur plusieurs industries, langues et juridictions",
+  "numbers.block1.title": "400+ services",
+  "numbers.block2.desc": "classés selon une taxonomie d'une cinquantaine de types",
+  "numbers.block2.title": "900+ documents",
+  "numbers.block3.desc": "publiées sous forme de brèves et de tweets.",
```
Comment (martinratinaud) [inline src/translations/fr/stats.json:23]:
```suggestion
  "numbers.block3.desc": "publiées sous forme de brèves et de tweets",
```

File: src/translations/fr/stats.mdx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## Impact qualitatif
+
+Un grande partie des effets d’Open Terms Archive est par essence non mesurable. Nous référençons dans cette section des cas marquants qui illustrent ces effets qui ont du sens lorsqu’ils sont contextualisés et le perdent lorsqu’ils sont réduits à un nombre, car leur impact systémique n’est pas linéaire selon leur occurrence.
+
+### Projets de loi
+
+Open Terms Archive a soutenu le [projet de loi TLDR Act](https://www.lifewire.com/the-tldr-act-could-help-you-make-sense-of-terms-of-service-agreements-5216643) (pour _Terms-of-service Labeling, Design, and Readability_) qui est une proposition de loi américaine co-portée par la députée Lori Trahan (D-MA-03), le sénateur Bill Cassidy (R-LA) et le sénateur Ben Ray Luján (D-NM) visant à rendre obligatoire la publication de résumés des conditions d’utilisation dans un format lisible par les machines.
```
Comment (martinratinaud) [inline src/translations/fr/stats.mdx:11]: Add links as in english version

**Review by martinratinaud — APPROVED** [review]
Nice congrats

Some other points:
- card_author should have a min height to prevent different display when author is on 2 lines
- container gray background is still not full width when on a big screen

File: src/pages/stats.tsx [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+              white={true}
+            ></ButtonBlock>
+          </ButtonBlockList>
+        </Container>
+      </Container>
+      <Container gridCols="12" gridGutters="11">
+        <CardList title={t('stats:instances.title')} small={true} centerTitle={true}>
+          <Card
+            title={t('stats:instances.contrib.title')}
+            subtitle={<Trans i18nKey={'stats:instances.contrib.subtitle'}></Trans>}
```
[the remark this answers is not in the document]
Comment (clementbiron) [reply src/pages/stats.tsx:69]: I need it to use a `<br />`

**Comment by clementbiron** [comment]
> card_author should have a min height to prevent different display when author is on 2 lines

I do not find the current display disturbing.

> container gray background is still not full width when on a big screen

This is normal ;)

Thanks for your review!

**Pull request merged**